ansible/roles/dev-desktop/tasks/dependencies.yml

```
@@ -136,7 +136,9 @@
 
 - name: Install Node
   apt:
-    name: nodejs
+    name: 
+      - nodejs
+      - npm
     state: present
     update_cache: yes
```

```suggestion
- name: Install Node
  apt:
    name: {{ item }}
    state: present
    update_cache: yes
  with_items:
   - node
   - npm
```
I think you need to wrap it in a loop (or just make two separate steps)

---

Just using a list works for the main dependency step, so I _think_ it's fine

https://github.com/rust-lang/simpleinfra/blob/9332e75d1ecdb6341b787a6b3c59df0bf908627d/ansible/roles/dev-desktop/tasks/dependencies.yml#L11-L18

---

ah, right. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_module.html#parameter-name